interfaces/builtin/wayland.go

```diff
@@ -109,6 +109,12 @@ owner /run/user/[0-9]*/wayland-[0-9]* rw,
 /etc/drirc r,
 `
 
+const waylandConnectedPlugAppArmorCore = `
+# Allow access to the Wayland compositor server socket of the slot snap
+owner /run/user/[0-9]*/###SLOT_SECURITY_TAGS###/wayland-[0-9]* rw,
+owner /###SLOT_SECURITY_TAGS###/wayland-[0-9]* rw,
```

Comment:
should we add this if we know it's not helping anyway? If I understand correctly this works only on 22.04 so far? I'm fine if that's the situation for now, but I don't think we should add rules that don't contribute